CoinDock-api/database/migrations/2022_05_17_163129_create_wallets_table.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->unsignedBigInteger('user_id');
             $table->foreign('user_id')->references('id')->on('users');
             $table->string('wallet_id');
-            $table->double('balance',2);
+            $table->double('USD_balance',10);
```

Comment:
Maintain sperate migration for the above change, please.

Reply:
double cannot be changed with separate migration.